### DISCUSSION TITLE
Scheduling / Recurring time based change of value

### DISCUSSION THREAD

**Discussion by mihaeldimitrovski** [frangoteam/FUXA#1038] [Q&A]
Hello,
Is it possible to change a value based on time, which re-occurs? I need to change the setpoint for a heat pump via modbus - each day at 5AM to go to a value of 52, and each day at 7AM to go to a value of 40.

Basically, what is the simplest way to create scheduled changes of values?

Thank you,
Mihael

**Comment by unocelli** [comment]
Hi,
you can configure a script with scheduling

![image](https://github.com/frangoteam/FUXA/assets/48684550/13ad0da0-c3d1-4bf5-b0ff-414f8d1ffc95)

**Reply by mihaeldimitrovski** [reply]
Hi @unocelli,
thank you for your reply.

'scheduling' seems greyed out?
Help is appreciated.

![image](https://github.com/frangoteam/FUXA/assets/155543365/ae7fcc2c-a7af-40e4-a69a-c00934f32977)

And this is the only value that I need changed e.g. each morning at 7AM:
![image](https://github.com/frangoteam/FUXA/assets/155543365/ad119aee-7cb1-4ee0-a5ae-33f976c92bb2)


thank you!
Mihael

**Reply by unocelli** [reply]
you need the last version, how you have installed, npm?

**Reply by unocelli** [reply]
1.1.16

**Reply by mihaeldimitrovski** [reply]
yes, npm

version that I have is 1.1.15-1324

**Reply by unocelli** [reply]
I have just published version 1.1.16

**Reply by mihaeldimitrovski** [reply]
Installing now.

**Reply by mihaeldimitrovski** [reply]
Great! Thank you! Scheduling is enabled.

Best regards.
